### DISCUSSION TITLE
logging in the new containers

### DISCUSSION THREAD

**Discussion by ahsanjav** [gadgetron/gadgetron#1131] [Q&A] [answered]
I know we discussed it this week but I totally missed that we don't log outputs to a log anymore or am I missing something ? 

/tmp/gadgetron.log is not created in the containers. We generally use that to monitor any issues incase recons fail - I am going to pipe outputs to a log but wanted to be sure that its the best solution at the moment ?

**Comment by hansenms** [comment, marked as answer]
Logs in the container go to stdout (as they should). They are not being piped to a file as well. Sending it to a file inside the container is a bit of a bad pattern, since it could fill up, etc. However, if you wanted to, you could override the `--entrypoint` and use `tee` to send to a file as well. 

The log file, btw, was a byproduct of using `supervisord` inside the container, which we have removed.
